pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -304,15 +304,18 @@ def _pod_progress_bar_force_stdout(self) -> None:
         if self.tpu_global_core_rank == 0 and int(os.getenv(xenv.TPUVM_MODE, 0)) == 1:
             print()
 
-    def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: _PATH) -> None:
+    def save_checkpoint(
+        self, checkpoint: Dict[str, Any], filepath: _PATH, storage_options: Optional[Any] = None
+    ) -> None:
         """Save model/training states as a checkpoint file through state-dump and file-write.
 
         Args:
             checkpoint: dict containing model and trainer state
             filepath: write-target file's path
+            storage_options: parameter for how to save to storage, passed to ``CheckpointIO`` plugin
         """
         # `xla_model.save` needs to be called on all ranks. It internally checks if the local rank is 0
-        self.checkpoint_io.save_checkpoint(checkpoint, filepath)
+        self.checkpoint_io.save_checkpoint(checkpoint, filepath, storage_options)
```

Comment:
Couldn't support `storage_options` for tpu_spawn strategy. We use the xla API instead of `fsspec` for checkpoint saving.

Reply:
But yeah, users should have the flexibility to subclass XLACheckpointIO plugin and use `storage_options`, the way they desire.

Reply:
> users should have the flexibility to subclass `XLACheckpointIO ` and use `storage_options`, the way they desire.

agree - in general, to subclass `CheckpointIO`